projects/Galene/example.nix

```diff
@@ -1,5 +1,22 @@
 { ... }:
 {
-  services.galene.enable = true;
-  services.galene.insecure = true;
+  services = {
+    galene = {
+      enable = true;
+
+      port = 8443;
+
+      tls = {
+        enable = true;
+        certificatePath = "Path/"
+        keyfile = "keyfile.pem"
```

Comment:
This file needs to exist, else the module will fail to evaluate. Can't we have `tls` off for now?